[2024-07-22T02:36:07.292+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-30T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:36:07.314+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: example_dynamic_task_mapping.add_one scheduled__2022-12-30T00:00:00+00:00 map_index=1 [queued]>
[2024-07-22T02:36:07.315+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:07.315+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-07-22T02:36:07.315+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-07-22T02:36:07.336+0000] {taskinstance.py:1300} INFO - Executing <Mapped(_PythonDecoratedOperator): add_one> on 2022-12-30 00:00:00+00:00
[2024-07-22T02:36:07.347+0000] {standard_task_runner.py:55} INFO - Started process 3346 to run task
[2024-07-22T02:36:07.368+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dynamic_task_mapping', 'add_one', 'scheduled__2022-12-30T00:00:00+00:00', '--job-id', '1231', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_dynamic_task_mapping.py', '--cfg-path', '/tmp/tmpwwbmzkqz', '--map-index', '1']
[2024-07-22T02:36:07.372+0000] {standard_task_runner.py:83} INFO - Job 1231: Subtask add_one
[2024-07-22T02:36:07.457+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 1231 for task add_one ([Errno 2] No such file or directory: '/opt/airflow/logs/dag_id=example_dynamic_task_mapping/run_id=scheduled__2022-12-30T00:00:00+00:00/task_id=add_one/map_index=1/attempt=1.log'; 3346)
[2024-07-22T02:36:07.500+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2024-07-22T02:36:07.594+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
